docs/connected_orgs.rst

```diff
@@ -1,14 +1,120 @@
+
+Sandboxes
+^^^^^^^^^
+For sandboxes, pass the ``--sandbox`` flag along with the org name.
+
+.. code-block:: console
+
+    $ cci org connect <org_name> --sandbox
+
+.. note:: The ``--sandbox`` flag can also be used for connecting a scratch org created externally to CumulusCI.
+
+Verify Your Connected Orgs
 --------------------------
+
+Run ``cci org list`` to see your org listed under the "Connected Org" table.
+
+    Example: A single persistent org connected to CumulusCI with the name ``devhub``.
+
+.. code-block:: yaml
+
+    $ cci org list
+
+    ┌Scratch Orgs──────────────┬─────────┬──────┬─────────┬──────────────┬────────┐
+    │ Name                     │ Default │ Days │ Expired │ Config       │ Domain │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ dev                      │         │ 7    │ X       | dev          │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ feature                  │         │ 1    │ X       | feature      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ prerelease               │         │ 1    │ X       | prerelease   │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ qa                       │         │ 7    │ X       | qa           │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ release                  │         │ 1    │ X       | release      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+
+    ┌Connected Orgs────┬──────────────────────────────┬────────────┐
+    │ Name   │ Default │ Username                     │ Expires    │
+    ├────────┼─────────┼──────────────────────────────┼────────────┤
+    │ devhub │         │ peter.gibbons@initech.devhub │ Persistent │
+    └────────┴─────────┴──────────────────────────────┴────────────┘
+
+Make sure that CumulusCI can login to the connected org.
+
+.. code-block:: console
+
+    $ cci org browser <org_name>
+
+
+
+Global Orgs
+-----------
+By default, ``cci org connect`` stores the OAuth credentials for connected orgs in a *project specific* keychain. These credentials ensure that an org connected in Project A's directory isn't available when you're working in Project B's directory.
+
+Connect an org and make it available to *all* CumulusCI projects on your computer by passing the ``--global-org`` flag.
+
+.. code-block:: console
+
+    $ cci org connect <org_name> --global-org
+
+
+
+Use a Custom Connected App
+----------------------------
+CumulusCI uses a preconfigured Connected App to authenticate to Salesforce orgs that use OAuth2. In most cases this works fine. To control the Connected App for specific security or compliance requirements, create your own Connected App and configure CumulusCI to use it when connecting to orgs.
```

Comment:
"In most cases this works fine." Bit of an odd sentence. What happens when it doesn't work? Do we have a link for troubleshooting?

Reply:
We meant that there are scenarios where it's not the right solution, not that it breaks intermittently. Examples of those scenarios are:
1. When running CumulusCI in a CI system and using the JWT Oauth2 flow to authenticate to an org (because this flow requires having a private key that was used to sign a certificate that is part of the Connected App configuration, and we can't provide this for the default Connected App)
2. The default Connected App allows any Salesforce user to approve the app, and do anything the user has permission to do. An organization may want to use their own Connected App configuration to enforce additional restrictions on things like who can use it, what they can do, and where they can connect from.

Reply:
@davisagli Is this documented anywhere? It's a really solid explanation.

Reply:
@justinvholt We do mention the need for creating a custom connected app in the section about running CumulusCI within a CI system using the JWT oauth2 flow. Feel free to incorporate these examples here to make the explanation clearer, if you think it fits.